nano/css/shared.css

```diff
@@ -721,3 +730,23 @@ th.bot {
 	background-color: black;
 	color: black;
 }
+
+/* [Tooltips of description] */
+.withToolTip .ImTooltip, .withToolTipInTooltip .ImTooltipInTooltip{
+	display			: none;
+	padding			: 10px;
+	margin-left		: 8px;
+	width			: 200px;
+	z-index			: 999;
+	background      : #717177;
+	border          : 4px solid #666;
+	border-radius   : 0px 5px;
+	color           : #d1d1dd;
+	text-align      : center;
+	white-space     : pre-line;
+	overflow: wrap;
+}
+.withToolTip:hover .ImTooltip, .withToolTipInTooltip:hover .ImTooltipInTooltip{
+	display			: block;
+	position		: absolute;
+}
```

Comment:
I tried to toy with tooltips on UI, but haven't figured out how to make it work there.
So I place it here for future use.